accounts/models.py

```diff
@@ -36,3 +37,11 @@ def email_user(self, subject, message, from_email=None, **kwargs):
     def __str__(self):
         return self.first_name + ' ' + self.last_name + ' (' + self.email + ')'
 
+
+class GoogleAccount(models.Model):
```

Comment:
Why are we creating a separate Model for this? There are Oauth backends available in `python-social-core`

Reply:
okay sir